CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+## How to contribute to retroachievements-composer
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+Please note we have a Pull Request Process, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+* Creating a new branch off of the develop branch
+* All changes to the code should have appropriate unit tests
+* All code should match PSR-2 style
+* Submit a pull request again develop as the base branch
+* All travis checks will need to pass
+#### **Did you find a bug?**
+
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/joestrong/retroachievements-composer/issues).
+
+
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/joestrong/retroachievements-composer/issues/new). Be sure to include a **title and clear description**, as much relevant information as possible, and a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+
+#### **Did you write a patch that fixes a bug?**
+
+
+* Open a new GitHub pull request with the patch.
+
+
+* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
+
+
+#### **Do you intend to add a new feature or change an existing one?**
+
+
+* Do not open an issue on GitHub until you have collected positive feedback about the change. GitHub issues are primarily intended for bug reports and fixes.
+
+
+#### **Do you have questions about the source code?**
+
+
+</br>
+
+
+RetroAchievements.org is a volunteer effort. We encourage you to pitch in and [join the team](https://github.com/joestrong/retroachievements-composer/graphs/contributors)!
```

Comment:
This is not for the RetroAchievements.org site itself, so can you change 'RetroAchievements.org' for 'This project'